### PR TITLE
add trigger button on PR page. 

### PR DESCRIPTION
**Pull request by avikalpg**
## Summary by CodeRabbit

- **New Features**
	- Introduced a button to trigger actions directly from GitHub and Bitbucket pull requests.
	- Enhanced user interaction with dynamic button states (e.g., "Triggering...", "Triggered") to provide feedback during operations.

- **Bug Fixes**
	- Improved handling of DOM changes to ensure the button appears correctly, even if the page structure changes.

- **Chores**
	- Updated the application version from 1.2.4 to 1.2.5 to reflect minor enhancements and bug fixes.

### CODE REVIEW

File: scripts/orchestrator.js [hunk 1]
```diff
@@ -51,6 +51,20 @@ const orchestrator = async (tabUrl, websiteUrl, userId) => {
...
+				const repoName = urlObj[4];
+				const prUrl = `https://github.com/${ownerName}/${repoName}/pull/${prNumber}`;
+
+				// Initial attempt to add the button
+				addTriggerButton('github', prUrl, websiteUrl);
+
+				// Set up a MutationObserver to watch for changes in the DOM
+				const observer = new MutationObserver(() => addTriggerButton('github', prUrl, websiteUrl));
+				observer.observe(document.body, { childList: true, subtree: true });
+			}
```
[the remark this answers is not in the document]
Comment (avikalpg) [reply scripts/orchestrator.js:67]: Where will I even add the observer.disconnect() code?

**Comment by avikalpg** [comment]
This pull request does not close #103

**Comment by avikalpg** [comment]
Here are the screen recordings of how this button looks:

1. GitHub (also demonstrating success): 
chrome_disY6vBOr3

2. Bitbucket (also demonstrating failure): 
image

**Review by MuskanPaliwal — APPROVED** [review]
Looks good to me

**Pull request merged**